src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -784,6 +785,62 @@ public void orderByAscFieldWithSpaceTest() throws IOException {
         Assert.assertEquals("The list is not in ascending order", sortedTestFields, testFields);
     }
 
+    @Test
+    public void testWhereWithBoolCondition() throws IOException {
+        JSONObject response = executeQuery(
+                    StringUtils.format("SELECT * " +
+                                    "FROM %s/account " +
+                                    "WHERE male = true ORDER BY age LIMIT 5",
+                            TestsConstants.TEST_INDEX_BANK)
+        );
+
+        JSONArray hits = getHits(response);
+        Assert.assertTrue(hits.length() == 4);
```

Comment:
usually it's better to use `assertEquals`, but even more better to use `assertThat(actualValue, equalTo(expectedValue))`. The second option is more verbose, but if the assertion fails, the output log message is much more descriptive.